### PR TITLE
[TC Components] -> [Vue UI]

### PR DESCRIPTION
**Pull request by TimoScheuermann**
# TC Components

A library of high-quality ready to use Vue.js components that will help you speed up your development workflow. Including 29 components and 4 layouts. Highly customizable.

Link: https://components.timos.design/
#### Checklist:

- [x] I have performed a self-review of submitted resource and its follows the guidelines of the project.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 ## Table of Contents
 
+- [Table of Contents](#table-of-contents)
```
[the remark this answers is not in the document]
Comment (TimoScheuermann) [reply readme.md]: removed it, one of my extensions automatically added it.

**Comment by TimoScheuermann** [comment]
> Also, add only one resource link to the site. Looks like you are pointing to subdomains from one website.

See this: [https://github.com/bradtraversy/design-resources-for-developers/pull/653#issuecomment-706499179_](https://github.com/bradtraversy/design-resources-for-developers/pull/653#issuecomment-706499179_)

File: readme.md [hunk 2]
```diff
@@ -12,24 +12,24 @@
...
 - [Design Systems & Style Guides](#design-systems--style-guides)
 - [Online Design Tools](#online-design-tools)
 - [Downloadable Design Software](#downloadable-design-software)
-- [Design Inspiration](#Design-inspiration)
-- [Image Compression](#Image-compression)
-- [Chrome Extensions](#Chrome-extensions)
+- [Design Inspiration](#design-inspiration)
+- [Image Compression](#image-compression)
+- [Chrome Extensions](#chrome-extensions)
 - [Others](#others)
```
Comment (Alaev) [inline readme.md:33]: revert this to the original state plase

**Review by Alaev — CHANGES_REQUESTED** [review]

File: readme.md (hunk 2 above)
Comment (TimoScheuermann) [reply readme.md]: done

**Review by Alaev — APPROVED** [review]

**Pull request merged**